express_relay/sdk/js/src/examples/simpleSearcherLimo.ts

```diff
@@ -112,41 +119,49 @@ class SimpleSearcherLimo {
       order.address,
       bidAmount,
       new anchor.BN(Math.round(Date.now() / 1000 + DAY_IN_SECONDS)),
-      this.chainId
+      this.chainId,
+      this.expressRelayConfig!.relayerSigner,
```

Comment:
Why it can hold an undefined value?

Reply:
since the initialisation doesn't happen in the constructor, it can be undefined. I can add a check though